packages/twenty-server/src/workspace/workspace-sync-metadata/reflective-metadata.factory.ts

```diff
@@ -94,7 +133,7 @@ export class MetadataParser {
         assert(
           fromObjectMetadata,
           `Object ${relation.fromObjectNameSingular} not found in DB 
-        for fromRelation defined in class ${objectMetadata.nameSingular}`,
+        for relation defined in class ${objectMetadata.nameSingular}`,
```

Comment:
If you don't like fromRelation can we keep at least the info like
```suggestion
        for relation FROM defined in class ${objectMetadata.nameSingular}`,
```
or something like that? to differentiate the direction?